### PR TITLE
Add & delete notes

### PR DESCRIPTION
**Pull request by jeetbhalani15**
##  Add & Delete Notes Functionality
-Added Add Note using API calls.
-Added Delete Note using API calls.

hard code some colors in CSS change it after applying dark mode.

### CODE REVIEW

File: Gravity-Notes/src/App.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { Login } from "./Pages/AuthWidget/Login/Login";
 import { SignUp } from "./Pages/AuthWidget/SignUp/SignUp";
 import { Homepage } from "./Pages/Homepage/Homepage";
 import { Landingpage } from "./Pages/Landing-page/Landing-page";
```
Comment (vikrantkode) [inline Gravity-Notes/src/App.js:10]: you can create `index.js` file for folders and export components from there.

File: Gravity-Notes/src/Components/Notes-card.js/Notecard.js [hunk 2]
```diff
@@ -5,14 +5,43 @@ import {BiImage} from "react-icons/bi"
...
 import {MdMoreVert} from "react-icons/md"
 import "./Notecard.css";
-export const Notecard = ({title, content, key})=>{
+import { useAuth } from "../../Contexts/Auth-context"
+import { useNotes } from "../../Contexts/NotesAction-context"
+import axios from "axios"
+
+
+
+
```
Comment (vikrantkode) [inline Gravity-Notes/src/Components/Notes-card.js/Notecard.js:14]: Remove unwanted spaces for better readability - Just Personal Suggestion (you can overlook it)

**Review by vikrantkode — APPROVED** [review]
- just some suggestions added. You can look into it.
- Otherwise all good

File: Gravity-Notes/src/Components/Notes-card.js/Notecard.css [hunk 3]
```diff
@@ -32,9 +32,10 @@
     outline: none;
 }
 .cardicon{
-    display: flex;
     background: #ffffff;
```
Comment (sahiba0915) [inline Gravity-Notes/src/Components/Notes-card.js/Notecard.css:35]: can use color variables here.

**Review by sahiba0915 — APPROVED** [review]
Added one suggestion , rest looks good, you used good variable names and comments which make code more readable, one thing remove unwanted space. rest good to go.

**Pull request merged**